packages/cozy-stack-client/src/CozyStackClient.js

```diff
@@ -268,14 +278,9 @@ class CozyStackClient {
         body = JSON.stringify(body)
       }
     }
+    clonedOptions.throwFetchErrors = true
```

Comment:
What is the purpose of this option if this is always set to true? 
Besides, I don't think we should make it true by default, this is quite a breaking change

Reply:
This is not where the breaking change is. This option is always set to `true` only in `fetchJSONWithCurrentToken` as this function was previously throwing errors (although the underlying call to `CozyStackClient.fetch` would also emit them).

Reply:
Yes but the vast majority our fetch calls go though `fetchJSON`, calling `fetchJSONWithCurrentToken`, calling `fetch`. Thus, forcing `throwFetchErrors = true` in `fetchJSONWithCurrentToken`, will break the current error emit behaviour

Reply:
Well yes, but I was expecting apps calling `fetchJSON` to actually catch the thrown errors (otherwise we would see broken behaviors all the time :thinking: )

Reply:
I could leave the `emit` call even when `throwFetchErrors` is `true` though and throw after that depending on the value of `throwFetchErrors`.

Reply:
> I could leave the emit call even when throwFetchErrors is true though and throw after that depending on the value of throwFetchErrors.

Sounds great!